web-ui/src/helpers/index.js

```diff
@@ -0,0 +1,6 @@
+export * from './celebration';
+export * from './checks';
+export * from './datetime';
+export * from './query-parameters';
+export * from './sanitizehtml';
+export * from './strings';
```

Comment:
This barrel allows us to import directly from `helpers`, any of the non-test files in the helpers folder.